P5/Source/Specs/att.datcat.xml

```diff
@@ -7,51 +7,221 @@ $Date$
+          valueDatcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545">NN</pos>
+      </gramGrp>
+    </egXML>
+    <p>Again, the issue of efficiency and redundancy of this type of interoperable markup demands
+      that the references to the particular data categories are provided either in the header of the
+      dictionary or at the level of ODD, in the element <gi>equiv</gi> (child of <gi>elementSpec</gi>), which constitutes an
+      equivalent of the <ident>att.datcat</ident> class.</p>
   </exemplum>
-  <remarks versionDate="2012-04-22" xml:lang="en">
-    <p>ISO 12620:2009 is a standard describing the data model and procedures for a Data
-      Category Registry (DCR). Data categories are defined as elementary descriptors in a
-      linguistic structure. In the DCR data model each data category gets assigned a
-      unique Peristent IDentifier (PID), i.e., an URI. Linguistic resources or preferably
-      their schemas that make use of data categories from a DCR should refer to them using
-      this PID. For XML-based resources, like TEI documents, ISO 12620:2009 normative
-      Annex A gives a small Data Category Reference XML vocabulary (also available online
-      at <ptr target="http://www.isocat.org/12620/"/>),
-      which provides two attributes, <att>dcr:datcat</att> and
-      <att>dcr:valueDatcat</att>.</p>
+  <exemplum xml:lang="en">
+    <p>The <att>targetDatcat</att> attribute is reserved for feature structure declarations, and is
+      analogous to the <att>targetLang</att> attribute of the <ident>att.pointing</ident> class, in that it describes the
+      object that is being referenced, rather than the referencing object.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
+      <fDecl name="POS" targetDatcat="http://hdl.handle.net/11459/CCR_C-396_5a972b93-2294-ab5c-a541-7c344c5f26c3">
+        <fDescr>part of speech (morphosyntactic category)</fDescr>
+        <vRange>
+          <vAlt>
+            <symbol value="NN" datcat="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545"/>
+            <symbol value="NP" datcat="http://hdl.handle.net/11459/CCR_C-1371_fbebd9ec-a7f4-9a36-d6e9-88ee16b944ae"/>
+            <!-- ... -->
+          </vAlt>
+        </vRange>
+      </fDecl>
+    </egXML>
+    <p>Above, the modeled feature uses <att>targetDatcat</att>, because if it used
+        <att>datcat</att> then we would be saying that the <gi>fDecl</gi> is an instance of the part-of-speech container data category,
+      whereas it is not — the <gi>fDecl</gi> models a container (<gi>f</gi>) that encodes a part
+      of speech. Note also that it is the <gi>f</gi> that is modeled above, not its values, which
+      are used as direct references to data categories, hence the use of <att>datcat</att> in the
+        <gi>symbol</gi> element.</p>
+  </exemplum>
+  
+  <remarks versionDate="2022-09-17" xml:lang="en">
```

Comment:
This entire `<remarks>` seems more important than a remark in a tagdoc should be. Seems to me it belongs in the main prose of the _Guidelines_. But to be honest, I am not sure exactly where it would go.

Reply:
I was thinking the same, and then I thought that it would simply be a nice step forward to start by having it all in a single place, and then be able to speculate about whether it should also sneak into the text of the Guidelines and where.